c2rust-analyze/src/main.rs

```diff
@@ -356,7 +356,7 @@ impl<'tcx> Debug for AdtMetadataTable<'tcx> {
     }
 }
 
-fn run(tcx: TyCtxt) {
+fn run<'tcx>(tcx: TyCtxt<'tcx>) {
```

Comment:
because the closure `assert_libc_cvoid` takes in a `Place` that has a lifetime parameter that's ambiguous otherwise (the compiler assumes its data escapes the closure)